asreview/webapp/src/ProjectComponents/ProjectPage.js

```diff
@@ -85,13 +104,62 @@ const ProjectPage = (props) => {
     }
   );
 
+  const isSimulating = () => {
+    return (
+      data?.mode === projectModes.SIMULATION &&
+      data?.projectInitReady &&
+      !data?.reviewFinished
```

Comment:
Oke, this is the upcoming status. 

I think we might need a discussion about whether mode and status should be in the Redux store. This makes this a lot easier.

Reply:
Perhaps when we have the `status` (`setup`, `running`, and `stopped`), this would be already resolved. Namely, the returned status `running` can replace `isSimulating` defined here.